play/PlayerRules/Basic Rules.txt

```diff
@@ -131,15 +131,27 @@ Feats		- Your character will gain special perks as you level up. This
 Player Stats:
 ==============================
 
-Each player has what are called Base Stats. An average person has a '5'
-for each stat, but an average player has a '6' . These stats are:
+Each player has what are called Base Stats. An average person has a '5' for
+each stat, but an average player has a '6'. Base Stats are split into Primary
+and Secondary stats. Primary stats are determined by your Base Stat points.
+Secondary stats are derived from several primary stats. For every point in a primary
+stat, a secondary stat point is awarded (if the primary and secondary stats are
+associated). The maximum number of points that can be put into a given secondary stat
+is determined by the quantity of points in the largest associated primary stat.
+These stats are as follows (secondary stat associations are indented):

```

Comment:
I guess we could try and call it, "carry strength", "endurance", "carry capacity", or "carry weight"
But I think "Carry Ability" is the most removed from common game terms. And I didn't want people to jump to the conclusion that this is some kind of universal every-object-has-weight system.